test.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+import winnowing
+
+
+class TestWinnowing(unittest.TestCase):
+    # testing for R language
+    def test1(self):
+        check = winnowing.Winnowing(file1= "/Users/aryangupta/Desktop/IIT_Bombay/MOSS-winnowing-seqMatcher/tests/test6.r",file2="/Users/aryangupta/Desktop/IIT_Bombay/MOSS-winnowing-seqMatcher/tests/test7.r",lang="r")
+        self.assertEqual(check.jaccardCheck(),0.9130434782608695)
+        self.assertEqual(check.plagiarismRate(),0.9130434782608695)
+
+    # testing for Python language
+    def test2(self):
+        check = winnowing.Winnowing(file1= "/Users/aryangupta/Desktop/IIT_Bombay/MOSS-winnowing-seqMatcher/tests/test2.py",file2="/Users/aryangupta/Desktop/IIT_Bombay/MOSS-winnowing-seqMatcher/tests/test3.py",lang="py")
```

Comment:
Path should be relative. Use `os` module to get the file path. These tests will fail in other systems. Change all the absolute paths to relative paths.